tests/contrib/pydantic_ai/test_pydantic_ai_llmobs.py

```diff
@@ -1,7 +1,8 @@
+from typing import TypedDict
+
 import mock
 import pydantic_ai
 import pytest
-from typing_extensions import TypedDict
```

Comment:
pydantic_ai tests require that `TypedDict` is imported from  `typing_extensions` if `python<3.12` is used. This is only needed for testing. 

We can revert the change in this file and then:
- Update this pydantic_ai test venv to install typing_extensions: https://github.com/DataDog/dd-trace-py/blob/main/riotfile.py#L3000. 
- Run this script to regenerate riot requirement files: https://github.com/DataDog/dd-trace-py/blob/main/docs/contributing-testing.rst#why-is-my-ci-run-failing-with-a-message-about-requirements-files

This should resolve this error: `'Please use `typing_extensions.TypedDict` instead of `typing.TypedDict` on Python < 3.12.',
                    code='typed-dict-version',`